### PR TITLE
Icons in the Worker Inspector

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description

Fun little change here, I've added icons to each component in the Worker Inspector and allow this to be set via a schema annotation! 

image

#### Tests

- [x] Standard library components with icons
- [x] Default icon for components
- [x] Annotations set the icon

#### Documentation

- [x] Changelog

Will also need to make a note of this in the Readme docs.

### CODE REVIEW

File: workers/unity/Packages/io.improbable.gdk.core/.schema/improbable/gdk/editor/worker_inspector.schema [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package improbable.gdk.editor;
+
+type ComponentIcon {
+    string icon_name = 1;
+}
```
Comment (zeroZshadow) [inline workers/unity/Packages/io.improbable.gdk.core/.schema/improbable/gdk/editor/worker_inspector.schema:5]: missing eol, is the editorconfig file not setup for schema?

File: workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/ComponentVisualElement.cs [hunk 2]
```diff
@@ -21,6 +23,25 @@ protected ComponentVisualElement()
...
+        protected void InjectComponentIcon(string iconName)
+        {
+            var iconContent = EditorGUIUtility.IconContent(iconName);
+            var iconVisualElement = new VisualElement();
+            iconVisualElement.style.backgroundImage = new StyleBackground((Texture2D) iconContent.image);
+            iconVisualElement.AddToClassList("component-icon");
+
+            // We want to inject the icon after the toggle icon and before the text.
+            var foldoutToggle = ComponentFoldout.Q<VisualElement>(className: "unity-toggle__input");
+            var children = foldoutToggle.Children().ToList();
```
Comment (zeroZshadow) [inline workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/ComponentVisualElement.cs:35]: This looks a little hacky, could be made simpler using `PlaceBehind`?

File: workers/unity/Packages/io.improbable.gdk.core/.schema/improbable/gdk/editor/worker_inspector.schema (hunk 1 above)
Comment (jamiebrynes7) [reply workers/unity/Packages/io.improbable.gdk.core/.schema/improbable/gdk/editor/worker_inspector.schema:5]: It is but you have to open it in an editor which supports `editorconfig` files, VSCode apparently, does not out of the box.

File: workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/ComponentVisualElement.cs (hunk 2 above)
Comment (jamiebrynes7) [reply workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/ComponentVisualElement.cs]: Turns out there's an `Insert(index, element)` method which I just... missed.. the first time around!

File: schema/playground/launcher.schema [hunk 3]
```diff
@@ -41,6 +42,8 @@ component Launcher {
     command ScoreIncreaseResponse increase_score(ScoreIncreaseRequest);
 }
 
+
```
Comment (jamiebrynes7) [inline schema/playground/launcher.schema:45]:
```suggestion
```

**Pull request merged**